mobile/examples/object_detection/ios/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+source 'https://github.com/CocoaPods/Specs.git'
+
+target 'ORTObjectDetection' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+  # Pods for ORTObjectDetection
+  pod 'onnxruntime-mobile-objc', '1.8.1'
```

Comment:
>, '1.8.1' [](http://example.com/codeflow?start=31&length=9)

now that we have a release version, should we always use the latest (i.e., not specify a version here)? maybe we can do that after setting up a CI test for this example, that can be done later